golang/lib/event/event_types.go

```diff
@@ -52,10 +53,12 @@ func NewShouldSendMetricsUserElectionEvent(didUserAcceptSendingMetrics bool) *Ev
 	return event
 }
 
-func NewCreateEnclaveEvent(enclaveId string) *Event {
+func NewCreateEnclaveEvent(enclaveId string, isSubnetworkingEnabled bool) *Event {
 	hashedEnclaveId := hashString(strings.TrimSpace(enclaveId))
+	isSubnetworkingEnabledStr := fmt.Sprintf("%v", isSubnetworkingEnabled)
 	properties := map[string]string{
-		enclaveIDPropertyKey: hashedEnclaveId,
+		enclaveIDPropertyKey:      hashedEnclaveId,
+		isSubnetworkingEnabledKey: isSubnetworkingEnabledStr,
```

Comment:
So you can dump whatever metadata you want in this thing? I guess it makes sense. Like tags in datadog

Reply:
Indeed! No db changes required. The thing will run alter table etc for you